common/types/activation.go

```diff
@@ -228,8 +223,7 @@ func (atx *ActivationTx) MarshalLogObject(encoder log.ObjectEncoder) error {
 		encoder.AddUint64("vrf_nonce", uint64(*atx.VRFNonce))
 	}
 	encoder.AddString("coinbase", atx.Coinbase.String())
-	encoder.AddUint32("pub_layer_id", atx.PubLayerID.Uint32())
-	encoder.AddUint32("epoch", uint32(atx.PublishEpoch()))
+	encoder.AddUint32("epoch", uint32(atx.PublishEpoch))
```

Comment:
NIT: Use the method you added 🙂 

```suggestion
	encoder.AddUint32("epoch", atx.PublishEpoch.Uint32())
```